### PR TITLE
CI improvements, plus make hw-latest the default

### PR DESCRIPTION
**Pull request by korran**
Please review each commit independently.

### CODE REVIEW

File: hw/fpga/fpga_configuration.tcl [hunk 1]
```diff
@@ -28,10 +27,10 @@ if {[info exists VERSION] == 0} {
 }
 
 # Path to rtl
-if {$HW_LATEST} {
-  set rtlDir $fpgaDir/../latest/rtl
+if {$RTL_VERSION} {
+  set rtlDir $RTL_DIR/../$RTL_VERSION/rtl
```
Comment (jlmahowa-amd) [inline hw/fpga/fpga_configuration.tcl]: This intermediate step is probably broken because RTL_DIR isn't set.

**Review by rusty1968 — DISMISSED** [review]

**Review by rusty1968 — APPROVED** [review]

File: hw/fpga/fpga_configuration.tcl (hunk 1 above)
Comment (korran) [reply hw/fpga/fpga_configuration.tcl]: Updated

**Review by jlmahowa-amd — APPROVED** [review]

File: hw/1.0/caliptra-rom-1.0.1-9342687.bin
Comment (jhand2) [inline hw/1.0/caliptra-rom-1.0.1-9342687.bin:1]: nit: Can you include a readme in this directory with a link to the release for any checked in binaries? This should make it easier to figure out exactly which commits are included.

**Review by jhand2 — APPROVED** [review]

**Review by mhatrevi — APPROVED** [review]

File: hw/1.0/caliptra-rom-1.0.1-9342687.bin
Comment (korran) [reply hw/1.0/caliptra-rom-1.0.1-9342687.bin:1]: Addressed in https://github.com/chipsalliance/caliptra-sw/pull/1441

**Pull request merged**